bootstrap/redis/producer/streams.go

```diff
@@ -129,10 +123,10 @@ func (es eventStore) Bootstrap(ctx context.Context, externalKey, externalID stri
 
 	if err != nil {
 		ev.success = false
+		return cfg, err
```

Comment:
No need to prevent publishing with the error indicator so we emit all events, even those unsuccessful.

Reply:
@dborovcanin I don't understand what this means 😅 . Should I remove the return statement, and let es.Add() run? I don't understand the reasoning

Reply:
Yes.